### ISSUE TITLE
Error when syncing from scratch at processing genesis block

### ISSUE THREAD

**Issue by diego-G** [LiskHQ/lisk-sdk#3292]
### Expected behavior

Peers are able to sync from scratch including genesis block.

### Actual behavior
```
0|node_0   | [ERR] 2019-04-09 14:06:11 | Failed to load blocks from the network. Error loading blocks: Failed to validate vote schema: Array is too long (101), maximum 33
0|node_0   | [dbg] 2019-04-09 14:06:11 | Loading blocks from the network
0|node_0   | [inf] 2019-04-09 14:06:11 | Received inbound request for procedure blocks
0|node_0   | [inf] 2019-04-09 14:06:11 | Responsed to peer request blocks
0|node_0   | [inf] 2019-04-09 14:06:11 | Responsed to peer request blocks

0|node_0  | [dbg] 2019-04-09 14:06:11 | Block processing failed - {"id":"6524861224470851795","err":"Failed to validate vote schema: Array is too long (101), maximum 33","module":"blocks","block":{"id":"6524861224470851795","version":0,"timestamp":0,"height":1,"numberOfTransactions":103,"totalAmount":"10000000000000000","totalFee":"0","reward":"0","payload
```
### Steps to reproduce

Run network tests with only 2 nodes.

### Which version(s) does this affect? (Environment, OS, etc...)
`feature/create_network_module`

**Comment by jondubois** [comment]
@shuse2 @pablitovicente Didn't we intentionally disable loading the genesis block from the network because of the 'RPC /blocks endpoint should return error if the lastBlockId provided is unknown.` issue?

**Comment by pablitovicente** [comment]
@jondubois Yes the genesis block is no longer returned; only blocks with height >= 2 and valid block ids

**Comment by jondubois** [comment]
@diego-G Maybe the real issue here is that we need to update nodes in the network tests to not need to fetch the genesis block from each other; they can each start with the same genesis block?

**Comment by shuse2** [comment]
This issue was due to the change in configuration, but it has been fixed with #3149
